### ISSUE TITLE
performance about fine-tuning TERA with output layer by minimizing a CTC loss

### ISSUE THREAD

**Issue by iehppp2010** [andi611/Self-Supervised-Speech-Pretraining-and-Representation-Learning#24]
Thanks your great work. I am a big fan of professor Hung-yi Lee, he's online course helps me a lot.

I have read the TERA paper, the ASR comparison experiments was conducted within the DNN/HMM framework. 
Have you ever tested the performance about fine-tuning TERA with output layer by minimizing a CTC loss?

**Comment by andi611** [comment]
Hi, thanks for reading our work!
Sadly I haven't tested with CTC.
